assets/packs/data_table/App.js

```diff
@@ -283,10 +293,12 @@ export function App({ ctx, data }) {
         ? content.data[col][row]
         : content.data[row][col];
 
+      const mappedCellData = mapCellData(kind, cellData)
```

Comment:
From the naming it's not clear why we map anything. For clarity, I would change the name above `cellData -> formattedValue` and here `mapCellData(kind, cellData) -> getCellData(kind, formattedValue)`.

Reply:
Fixed